src/edu/ucsb/nceas/metacat/admin/upgrade/HashStoreUpgrader.java

```diff
@@ -467,17 +519,29 @@ private void convert(Path path, String finalId, SystemMetadata sysMeta, String c
         try (InputStream sysMetaInput = convertSystemMetadata(sysMeta)) {
             metadata = converter.convert(path, finalId, sysMetaInput, checksum, algorithm);
         } catch (NonMatchingChecksumException e) {
-            logMetacat.error("Cannot move the object " + finalId + "to hashstore since "
-                                + e.getMessage());
-            writeToFile(finalId, nonMatchingChecksumWriter);
+            logMetacat.info("Continue to move the object " + finalId + "to hashstore even though "
```

Comment:
Hem. The whole block looks like:
```
try (InputStream sysMetaInput = convertSystemMetadata(sysMeta)) {
            metadata = converter.convert(path, finalId, sysMetaInput, checksum, algorithm);
        } catch (NonMatchingChecksumException e) {
            logMetacat.info("Continue to move the object " + finalId + "to hashstore even though "
                                + "its checksum in the system metadata didn't match Hashstore's "
                                + "calculation: " + e.getMessage());
            try {
                metadata =
                    reConvertUnMatchedChecksumObject(path, finalId, sysMeta, e.getHexDigests(),
                                                     nonMatchingChecksumWriter);
            } catch (Exception ee) {
                logMetacat.error("Cannot reconvert unMatchedChecksum object " + finalId + " since "
                                     + e.getMessage());
                writeToFile(finalId, e, generalWriter);
                return;
            }
        }
```
So this log statement just indicates that Metacat will start another route. The operator doesn't need this statement to take an action. They need to read the info file to take an action - sending the file to us. So do you think we need it `warn`?